lib/sql.js

```diff
@@ -1207,13 +1243,18 @@ SQLConnector.prototype.buildOrderBy = function(model, order) {
   if (typeof order === 'string') {
     order = [order];
   }
+  const props = self.getModelDefinition(model).properties;
   const clauses = [];
   for (let i = 0, n = order.length; i < n; i++) {
     const t = order[i].split(/[\s,]+/);
+    const key = t[0];
+    if (!props[key]) {
+      throw new Error('Unknown property ' + key + ' is used for model ' + model);
```

Comment:
Ditto, let's add a feature flag allowing users to get the old behavior. 

I think it's ok to have a single feature flag covering all different places touched by this pull request.

It's also important to cover both setups in our test suite: the old behavior (the feature flag is enabled) and the new behavior (the feature flag is not defined in model settings).

Reply:
Done